src/main/java/it/trade/model/reponse/TradeItPreviewStockOrEtfOrderResponse.java

```diff
@@ -19,17 +15,32 @@
     @Expose
     public String orderId;
 
+    @SerializedName("warnings")
+    @Expose
+    public List<Warning> warnings = new ArrayList<>();
+
+    /**
+     * @deprecated Use warnings.
```

Comment:
Can we get rid of this or is this still returned by the server?

Reply:
If we rip it out immediately anyone upgrading will see compilation errors.

Reply:
Yes, but we want them to update their code to use these new fields? Will they update their code if they don't see compilation errors?

Reply:
They'll see a warning when compiling which should remind them. It's better to give integrators time to migrate as opposed to rip something out from underneath them.